### ISSUE TITLE
Exchange Rate REST API

### ISSUE THREAD

**Issue by steven-sheehy** [hashgraph/hedera-mirror-node#3579]
### Problem

The mirror node should expose an exchange rate REST API so users can estimate gas price or exchanges can estimate cost in USD.

### Solution

* A new exchange rate REST API `/api/v1/network/exchangerate` will be added that returns the [exchange rate](https://github.com/hashgraph/hedera-protobufs/blob/main/services/exchange_rate.proto) network file.
* Retrieve `file_data` for file ID `0.0.112`. Extract the contract bytecode [logic](https://github.com/hashgraph/hedera-mirror-node/blob/55599ea2b6a5ab460f1c1f800d1f3f506a6e2c57/hedera-mirror-rest/controllers/contractController.js#L70) to a separate fileService.
* Cache max-age should be set to 60s or more since file only updates every hour.
```json
{
  "current_rate": {
    "cent_equivalent": 596987
    "expiration_time": 1649689200
    "hbar_equivalent": 30000
  },
  "next_rate": {
    "cent_equivalent": 594920
    "expiration_time": 1649692800
    "hbar_equivalent": 30000
  },
  "timestamp": "1649689200.123456789"
}
```
Query Parameters:

- `timestamp` - Show the exchange rate as of a certain timestamp or timestamp range. Supports `eq`, `gt`, `gte`, `lt`, `lte` operators, but note that it will only return the most recent exchange rate within the given timestamp range.

**Comment by Kalina-Todorova** [comment]
@steven-sheehy Do we already have this in the DB and if yes - could you point us to the table?

**Comment by steven-sheehy** [comment]
@Kalina-Todorova I've updated the description to note the data is available in `file_data` at file ID `0.0.112`.
